e2e/alerting_test.go

```diff
@@ -17,11 +18,17 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const (
+	MultiDirectiveTopicArn = "arn:aws:sns:us-east-1:000000000000:multi-directive-test-topic"
```

Comment:
Can we add a comment stating these are localstack specific values to simulate aws arns?